keps/sig-api-machinery/1040-priority-and-fairness/README.md

```diff
@@ -394,7 +394,12 @@ list of queue indices excluding I[0].  I[2] is the A[2]’th entry in
 the list of queue indices excluding I[0] and I[1].  And so on.
 
 The lengths of the queues identified by I[0], I[1], … I[H-1] are
-examined, and the request is put in one of the shortest queues.
+examined, and the request is put in one of the queues holding the
+least amount of work.  Originally this was just a matter of examining
+queue length.  With the generalizations for width and extra latency,
+the work in a queue is the sum of the work in its waiting requsts.
+The work in a request is the product of its width and its total
+estimated execution duration (including extra latency).
```

Comment:
FTR - This is already mentioned in the KEP below too:

https://github.com/kubernetes/enhancements/blame/master/keps/sig-api-machinery/1040-priority-and-fairness/README.md#L1268-L1275

But I'm obviously not opposed to putting it here too.